build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group = 'cloud.eppo'
-version = '1.0.0-SNAPSHOT'
+version = '2.0.0-SNAPSHOT'
```

Comment:
i've already released v1.0.0 so setting this to the potential next version snapshot